tests/unit/processor/pre_detector/test_pre_detector_rule.py

```diff
@@ -1,4 +1,5 @@
-from logprep.filter.lucene_filter import LuceneFilter
+# pylint: disable=missing-docstring
+# pylint: disable=protected-access
 
 import pytest
 
```

Comment:
You have removed the `pytest.importskip` in other rule tests, can we get rid of it here too?